.release.json

```diff
@@ -1,9 +1,9 @@
 {
   "source": {
     "repo_url": "https://github.com/onfido/onfido-openapi-spec",
-    "short_sha": "01ed1b5",
-    "long_sha": "01ed1b5fe0f7490bfce54816504a9fec13b33862",
-    "version": "v3.1.0"
+    "short_sha": "40b86a1",
+    "long_sha": "40b86a1c8ec8687c15514470f60428b3e7633968",
+    "version": "v3.2.0"
   },
-  "release": "v3.3.0"
+  "release": "v4.0.0"
```

Comment:
We did a minor for the other libraries, any reason why java lib gets a major?

Reply:
Good point, we went for a major update here because unfortunately the change in java is much more impacting (we don't have a UUID object anymore but a String one, therefore is a breaking change according to policy).

In other languages the impact of the change is much smaller, so we went for a minor update.

Reply:
Good point 👍